config/locales/server.en.yml

```diff
@@ -31,3 +32,22 @@ en:
 
       If you're interested, click the link below:
       [%{topic_link}](%{topic_link})
+  pending_assigns_reminder:
+    title: "You have %{pending_assignments} pending assignments"
+    body: |
+      You currently have [%{pending_assignments} pending assignments](%{assignments_link}).
+
+      %{newest_assignments}
+      %{oldest_assignments}
```

Comment:
I would only have N oldest with a (7 more) link at the bottom.  Keep it to say 10 + an optional N more link.

Reply:
Could you please explain? I'm not sure I'm understanding your point.

Reply:
Its ok lets just try this format for now.